NitroxClient/MonoBehaviours/MultiplayerSeaMoth.cs

```diff
@@ -24,9 +24,10 @@ protected override void Awake()
 
         protected void Update()
         {
+            // Clamp volume between 0 and 1 (nothing or max). Going below 0 turns up volume to max.
             float distance = Vector3.Distance(Player.main.transform.position, transform.position);
-            rpmSound.GetEventInstance().setVolume(1 - distance / radiusRpmSound);
-            revSound.GetEventInstance().setVolume(1 - distance / radiusRevSound);
+            rpmSound.GetEventInstance().setVolume(Mathf.Clamp01(1 - distance / radiusRpmSound));
```

Comment:
Would do the same in `NitroxServer/Communication/Packets/Processors/PlayFMODAssetProcessor.cs`.